android/package.json

```diff
@@ -19,13 +19,13 @@
 		"mode": "release"
 	},
 	"minSDKVersion": "16",
-	"compileSDKVersion": "26",
+	"compileSDKVersion": "27",
 	"vendorDependencies": {
-		"android sdk": ">=23.x <=25.x",
-		"android build tools": "26.x",
-		"android platform tools": "26.x",
+		"android sdk": ">=23.x <=27.x",
+		"android build tools": "<=26.x",
```

Comment:
Can you update build-tools version to "27.x" please?
Or perhaps it should be ">=25.x <=27.x".
You said v25 is the max version IntelliJ supports, right?